src/ansys/systemcoupling/core/adaptor/impl/injected_commands.py

```diff
@@ -31,48 +32,76 @@
 from .types import Container
 
 
+# We cannot import Session directly, so define a protocol for typing
+# We mainly use it as a means of accessing the "API roots".
+class SessionProtocol(Protocol):
+    case: Container
+    setup: Container
+    solution: Container
+    _native_api: NativeApi
```

Comment:
Just a small comment, does this get used or could it be removed?

Reply:
It will be used once the chart PR has gone in. Similarly the `download_file` method below.

Reply:
That makes sense! I was also wondering about the download_file method, thanks for the explanation. I'll resolve this comment.